docs/website/docs/plus/ecosystem/ms-sql.md

```diff
@@ -47,116 +36,51 @@ ENABLE CHANGE_TRACKING
-
-#### Understanding the Change Tracking Query
-
-The incremental loading process uses a SQL query that joins the CHANGETABLE function with the source table to fetch the latest changes. Here’s a simplified version of the query:
-
-```sql
-SELECT
-    [Columns],
-    ct.SYS_CHANGE_VERSION AS _dlt_sys_change_version,
-    CASE WHEN ct.SYS_CHANGE_OPERATION = 'D' THEN 'DELETE' ELSE NULL END AS _dlt_deleted
-FROM
-    CHANGETABLE(CHANGES [YourSchemaName].[YourTableName], @last_version) AS ct
-    LEFT JOIN [YourSchemaName].[YourTableName] AS t
-        ON ct.[PrimaryKey] = t.[PrimaryKey]
-WHERE
-    ct.SYS_CHANGE_VERSION > @last_version
-ORDER BY
-    ct.SYS_CHANGE_VERSION ASC
-```
-
-- *CHANGETABLE*: Retrieves changes for the specified table since the last tracking version.
-- **Join with Source Table**: The join retrieves the current data for the changed rows.
-- *SYS_CHANGE_VERSION*: Used to track and order changes.
-- *_dlt_deleted*: Indicates if a row was deleted.
-
-**Note**: Since the query joins with the production table, there may be implications for locking and performance. Ensure your database can handle the additional load, and consider isolation levels if necessary.
-
-
-## Step-by-Step Recommended Usage
-
-1. Set up `dlt` project.
-You can follow the documentation for `sql_database` in order to setup credentials:
-
-https://dlthub.com/docs/devel/dlt-ecosystem/verified-sources/sql_database/setup
-
-You'll also need a working ODBC driver:
-
-https://dlthub.com/docs/dlt-ecosystem/destinations/mssql
-
-2. Set Up the Pipeline
-
-```py
-import dlt
+1. **Initial Full Load**: Use the `sql_table` function to perform a full backfill of your table data.
+2. **Incremental Loading**: Use the `create_change_tracking_table` function to load incremental changes using SQL Server's Change Tracking.
 
-pipeline = dlt.pipeline(
-    pipeline_name='sql_server_sync_pipeline',
-    destination='your_destination',
-    dataset_name='destination_dataset',
-)
-```
+This approach ensures that you have a complete dataset from the initial load and efficiently keep it updated with subsequent changes.
 
-2. Configure the Database Connection
+### Initial Full Load
 
-In our example we create an explicit SQLAlchemy Engine. This is not required, you are free to
-follow the instructions to create credentials in `secrets.toml` (or env variables) instead: 
+Get the Change Tracking version **before you execute initial load** to make
+sure you do not miss any updates that may happen during it. This may result in
+"replaying" a few changes that happen during the load, but this will not have any
+impact on the destination data due to the `merge` write disposition. 
 
 ```py
+from dlt_plus.sources.mssql import get_current_change_tracking_version
 from sqlalchemy import create_engine
 
 connection_url = "mssql+pyodbc://username:password@your_server:port/YourDatabaseName?driver=ODBC+Driver+18+for+SQL+Server&TrustServerCertificate=yes"
```

Comment:
Maybe a note for the future, it's not very nice to ask user to define the credentials twice and directly in code. There is an alternative, where user only specify the credentials in toml/env and then get them:

```
import dlt
connection_url = dlt.secrets.get("destination.sql_database.credentials").to_native_representation()
```
maybe not exact this code, but something close, need to check

Reply:
yes, future